### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by jamshid-net**

### CODE REVIEW

**Comment by Jamshid-DotNet** [comment]
some changes

File: UserInterface/obj/project.nuget.cache [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
```
Comment (Jamshid-DotNet) [inline UserInterface/obj/project.nuget.cache:1]: 2

File: UserInterface/obj/project.assets.json [hunk 2]
```diff
@@ -298,17 +298,17 @@
         "net7.0": {
```
Comment (Jamshid-DotNet) [inline UserInterface/obj/project.assets.json:298]: 2

**Review by jamshid-net — COMMENTED** [review]
reviewed

**Review by jamshid-net — COMMENTED** [review]
reviewed changes

**Review by jamshid-net — COMMENTED** [review]
some changes

File: Admin/obj/Admin.csproj.nuget.dgspec.json [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {
   "format": 1,
```
Comment (Jamshid-DotNet) [inline Admin/obj/Admin.csproj.nuget.dgspec.json:2]: f

**Pull request closed without merging**